### PR TITLE
[WNMGDS-2984] Update Pagination Styling for Medicare

### PR DESCRIPTION
**Pull request by jack-ryan-nava-pbc**
## Summary

- [Ticket](https://jira.cms.gov/browse/WNMGDS-2984)
- Changes:
- Added an override to the base.scss for the medicare child design system
- Should set the button font family to Montserrat for the Pagination elements.

## How to test

1. Fire it up locally and confirm in Storybook/Docsite that the buttons in the Pagination component are indeed using the font family Montserrat
2. Or check out this [demo branch](https://cmsgov.github.io/design-system/branch/jryan/wnmgds-2984-add-in-new-pagination-styling)

## Checklist

- [X] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [X] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [X] Selected appropriate `Impacts`, multiple can be selected.
- [X] Selected appropriate release milestone

### CODE REVIEW

File: packages/ds-medicare-gov/src/styles/_base.scss [hunk 1]
```diff
@@ -115,3 +115,13 @@ svg {
...
+
+/**
+ * Pagination overrides
+ */
+
+.ds-c-pagination {
+  .ds-c-button {
+    font-family: var(--font-family-heading);
+  }
+}
```
Comment (pwolfert) [inline packages/ds-medicare-gov/src/styles/_base.scss:127]: Is there a way to do this without CSS overrides? We spent years trying to get CSS overrides _out_ of the Medicare theme and rely on tokens instead.

File: packages/ds-medicare-gov/src/styles/_base.scss [hunk 2]
```diff
@@ -115,3 +115,13 @@ svg {
   line-height: 20px;
   margin-bottom: $spacer-1;
 }
+
+/**
+ * Pagination overrides
+ */
+
+.ds-c-pagination {
+  .ds-c-button {
```
Comment (kim-cmsds) [inline packages/ds-medicare-gov/src/styles/_base.scss:124]: NB: it looks like there's a class name "ds-c-pagination__nav" for the pagination buttons that could make this rule a a bit simpler

**Review by kim-cmsds — APPROVED** [review]

File: packages/ds-medicare-gov/src/styles/_base.scss (hunk 2 above)
Comment (jack-ryan-nava-pbc) [reply packages/ds-medicare-gov/src/styles/_base.scss:124]: It looks like that class name is only used for the chevron icons, and not the page numbers.

File: packages/ds-medicare-gov/src/styles/_base.scss (hunk 1 above)
Comment (pwolfert) [reply packages/ds-medicare-gov/src/styles/_base.scss]: Looks good now 👍

**Comment by jack-ryan-nava-pbc** [comment]
Noting to self: sync tokens to Figma after this is merged in.

**Pull request merged**